src/engine/resources.ts

```diff
@@ -735,6 +742,21 @@ export function tryPlayApriling(modifier: string): void {
   }
 }
 
+export function photoboothEffect(modifier: string): void {
```

Comment:
We shouldn't do this in the same way as Apriling, since we can't switch back-and-forth as often. Probably better to stick to -combat and treat it similarly to submarine sprints. https://github.com/Kasekopf/looprobot/blob/d611e5144b3cca42b0a8059141d2661cc905bad0/src/engine/moods.ts#L78